articles/article-119-supervised-learning.md

```diff
@@ -0,0 +1,36 @@
+<!-- BEGIN TITLE -->
+# How do we now, how good the solution was
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+If we try to solve a problem, we always appreciated if we get feedback on how good we where and this is the same case for a program.
+In supervised learning we first let a program solve  problems,
+where we do now the results, like for example finding the labels of different points, so called classification. 
+Through this training we know how good the solutions of the program are and can adjust the program for efficiency,
+and can then use it for the same kind of problem where we do not know the solution. 
+
+A downside of this method is that it needs a lot of known information and is prone to overfitting,
+where we build the program so good to our known problem, that it does not solve the problem in general and only in these specific cases. 
+
+<!-- END BODY -->
+
+
+## Figure
+* A simple classification graph, some points and then the separating line (maybe to difficult for understanding)
+* A teacher computer, who teachs some smaller computer with a blackboard
+
+
+
+<!-- BEGIN OPTIONAL -->
+## Remark
+* Remove the overfitting part.
```

Comment:
Please don't put review notes into the optional text section. Instead, add them as a comment to the pull request.